src/thunderbots/external/g3log.BUILD

```diff
@@ -0,0 +1,38 @@
+# Description:
+#   G3log is a logging library that allows us to publish at multiple logging levels,
+#   log to custom sinks, etc.
+
+# G3log uses cmake to generate some files, so we need to run it
+# before we build
+genrule(
+    name = "g3log_cmake",
```

Comment:
I don't see anything depending on this genrule. Shouldn't the cc_library depend on it?

Reply:
Nevermind I see the header dependency